server/plugin/plugin.go

```diff
@@ -166,6 +168,9 @@ func (p *Plugin) OnConfigurationChange() (err error) {
 	p.updateEnv(func(e *Env) {
 		p.initEnv(e, pluginURL)
 
+		// TODO: replace with proper logic
+		e.EncryptedStore = true
```

Comment:
Probably good to address this in this PR or create a ticket for this

Reply:
Pending #268

Reply:
I have made it so the google calendar plugin uses the encrypted store while the mscalendar doesn't until we have a decission on what to do for mscalendar users.